roles/kubernetes-apps/container_engine_accelerator/nvidia_gpu/defaults/main.yml

```diff
@@ -6,5 +6,8 @@ nvidia_gpu_gtx_base_url: http://us.download.nvidia.com/XFree86/Linux-x86_64/
 nvidia_gpu_flavor: tesla
 nvidia_url_end: "{{nvidia_driver_version}}/NVIDIA-Linux-x86_64-{{nvidia_driver_version}}.run"
 nvidia_driver_install_container: false
+nvidia_driver_install_centos_container: atzedevries/nvidia-centos-driver-installer:2
+nvidia_driver_install_ubuntu_container: gcr.io/google-containers/ubuntu-nvidia-driver-installer@sha256:eea7309dc4fa4a5c9d716157e74b90826e0a853aa26c7219db4710ddcd1ad8bc
```

Comment:
I suggest to add those settings in `inventory/sample/group_vars/k8s-cluster/` as well to let people know that they can modify those settings.